values.yaml

```diff
@@ -246,7 +246,7 @@ imagePullSecrets:
 updateStrategy:
   type: RollingUpdate
 
-# debug, info, warning, error or fatal
+# debug, info, warn or error
```

Comment:
@utkudarilmaz Thank you for pointing out this.
The values of log level are different for different components.
The `debug`, `info`, `warning`, `error` and `fatal` work for `core` and the registry only supports `debug`, `info`, `warn` and `error`. So I think we can do some convert for registry